src/markdown.cpp

```diff
@@ -2854,6 +2854,9 @@ size_t Markdown::Private::writeBlockQuote(std::string_view data)
       }
       else
       {
+        curLevel=level;
+        i = indent;
+        if (txt[0] == '#') break; // begin of markdown section command
```

Comment:
looks like there is a risk we get here while `txt` is empty and since it is a string_view `txt[0]` can then trigger an out-of-bounds assert (on some systems).

Reply:
Indeed.

Solved some corner cases as well
```
> text
##NO header

##Second test on no header
```

and
```
> text
 ## text
```